src/main/java/evaluation/ParameterSearch.java

```diff
@@ -251,7 +252,7 @@ public static void runMultiNTBEA(NTupleSystem landscapeModel, String[] args) {
         Pair<Pair<Double, Double>, double[]> bestResult = new Pair<>(new Pair<>(Double.NEGATIVE_INFINITY, Double.NEGATIVE_INFINITY), new double[0]);
         for (int mainLoop = 0; mainLoop < repeats; mainLoop++) {
             landscapeModel.reset();
-            Pair<Double, Double> r = runNTBEA(null, evaluator, searchFramework, iterationsPerRun, iterationsPerRun, evalGames, verbose);
+            Pair<Double, Double> r = runNTBEA(null, evaluator, searchFramework, iterationsPerRun, 500, evalGames, verbose);
```

Comment:
This seems like a backwards change of variable to hard-coded number, any reason for it?

Reply:
Oops - left in from debugging. Fixed.